nsenter/nsexec.c

```diff
@@ -60,6 +60,62 @@ static int clone_parent(jmp_buf * env)
 	return child;
 }
 
+void nsinit_pid()
```

Comment:
i tried that but the end result wasnt as clean. Since spawning an init process with a known pid namespace and exec a new process inside another container are two very different operations , i think that having two functions describes the intent better.